src/styles/_variables.scss

```diff
@@ -30,3 +30,11 @@ $dark-text: $dark-rock;
 $warning: $yellow;
 $error: $red;
 $accent: $medium-green;
+
+$logo: #282828;
```

Comment:
Move this hex code to a color variable and let's rethink what we are doing with this name. Is the logo that same color everywhere or is it header specific?